recipes/stripnet/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "stripnet" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stripnet-{{ version }}.tar.gz
+  sha256: 66c3719a7ad6bbcc4110de6ef2ee124ca7320d8db12388b31b6eec35d0dcba42
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.9
+  run:
+    - python >=3.6,<3.9
+    - bertopic ==0.9.4
+    - ipywidgets ==7.6.5
+    - networkx ==2.6.3
+    - numpy ==1.22.0
+    - pandas ==1.3.5
+    - plotly ==5.5.0
+    - pyvis ==0.1.9
+    - scikit-learn ==1.0.2
+    - sentence-transformers ==2.1.0
```

Comment:
I feel these constrains are overly restrictive and will severely impact the solvability of the conda environments. In my opinion the runtime dependencies should be as little constrained as possible (ABI/API compatible). I understand this might be useful for reproducibility of exact results, but there are better solutions for that (conda-lock files, for example). 

Could you elaborate a bit? Thanks!

Reply:
I absolutely agree with you on the too strict requirements. But these are the `setup.py:install_requires` and my understanding is that run-requirements needs to be the same as install_requires. Is it?

I would prefer to use `>=` instead of `==`, wherever possible.